support/BRKey.c

```diff
@@ -364,7 +372,84 @@ size_t BRKeyLegacyAddr(BRKey *key, char *addr, size_t addrLen, BRAddressParams p
     return addrLen;
 }
 
-// signs md with key and writes signature to sig
+// convert signature from DER to raw encoding
+int BRSignatureDERToRaw (const uint8_t *derSig, size_t derSigLen, uint8_t *rawSig, size_t rawSigLen)
+{
+    const int sigLen = 64;
+    const int size = 32;
+    
+    if (NULL == rawSig) {
+        return sigLen;
+    } else if (rawSigLen < sigLen) {
+        return 0;
+    }
+    
+    //Skip 0x30
+    int offset = 1;
+    if (derSig[1] == 0x81) {
+        //Skip sign
+        offset++;
+    }
+    
+    //Convert to unsigned. Should match DER length - offset
+    int encodedLength = derSig[offset++] & 0xff;
+    assert(encodedLength == derSigLen - offset);
```

Comment:
I think we want to fail out (maybe in addition to an assert) since the assert may be a NOP depending on how this source is built.